shared/folders/files/index.js

```diff
@@ -24,8 +27,10 @@ type State = {
 class Files extends Component<void, Props, State> {
   state: State;
 
-  _checkFolderExistance (props) {
-    // TODO (AW): make a more user friendly response for when the folder they were hoping to look at has been removed
+  _checkFolderExistence (props) {
```

Comment:
These spelling fixes were probably intended for the previous commit. Probably fine to just squash 'em.

![sorry](http://s2.quickmeme.com/img/19/19e90aca81517dc837250c7b54ddc5240d03396958d41936500d5585d3baa5cc.jpg)